### ISSUE TITLE
Allow for multiple target variables to be generated

### ISSUE THREAD

**Issue by BrendanSchell** [FeatureLabs/compose#100]
In the case where I want to generate multiple target variables (for the same data slices), I currently have to loop through and join the tables together. It would be nice to be able to generate multiple target variables from a list of functions all at once.

**Comment by jeff-hernandez** [comment]
Hi @BrendanSchell,

Thanks for the feature request! If I understand correctly, this will let us apply multiple labeling functions to each data slice. For example, if we create a label maker using a list of labeling functions.

```python
lm = cp.LabelMaker(
    ...
    labeling_function=[a, b, c],
)
```

Then, we can run the search and the returned labels will already be joined together.

```python
lt = label_maker.search(...)
```

```
      target_id         cutoff_time       a      b       c
id                                            
0             1 2014-01-01 00:45:30    True   False   True
1             1 2014-01-01 00:46:35   False    True   True
```

Is this correct?

**Comment by BrendanSchell** [comment]
Yep, that's exactly what I was thinking!

**Comment by jeff-hernandez** [comment]
Nice, thanks for the confirmation! I will keep this issue updated on the progress.

**Comment by jeff-hernandez** [comment]
Hi @BrendanSchell,

This feature is available in the latest release 0.3.0! Let me know if you have any questions or feedback. Thanks again for the feature request!
